### ISSUE TITLE
JSON Web Tokens (JWT)

### ISSUE THREAD

**Issue by theganyo** [apigee-127/volos#20]
See: http://tools.ietf.org/html/draft-ietf-oauth-jwt-bearer

**Comment by borisirota** [comment]
@theganyo any estimation about this one ?

**Comment by theganyo** [comment]
No. It's not currently prioritized, but I'm interested in your expectations and use cases for this. That could help me prioritize this feature. Thanks!

**Comment by borisirota** [comment]
Authenticate users for a web app - nothing else in my mind :)

**Comment by saaqibz** [comment]
+1

**Comment by firefoxNX** [comment]
+1

**Comment by tanzim** [comment]
+1 
One nice use case is server to server APIs as used by Google. See: https://developers.google.com/identity/protocols/OAuth2ServiceAccount

**Comment by theganyo** [comment]
Thanks for the interest! I'll try to get some time to look at this. :)

**Comment by peraxel** [comment]
+1
